js/ui/scheduler/appointmentPopup/popup.js

```diff
@@ -176,6 +176,14 @@ export class AppointmentPopup {
         );
     }
 
+    _setTimeZonesVisibility() {
+        if(this.scheduler.getEditingConfig().allowTimeZoneEditing) {
```

Comment:
`this.scheduler` is a group of options for the Scheduler appointment popup and form. It'll be cool to rename this variable to the `this.config` e.g.